plugins/jobs/src/js/components/form/helpers/JobParsers.ts

```diff
@@ -13,6 +14,10 @@ import {
 } from "./JobFormData";
 import { schedulePropertiesCanBeDiscarded } from "./ScheduleUtil";
 
+// Array<[string, T]> => Record<string, T>
```

Comment:
Not intended I think.

Reply:
actually i thought it may be a good documentation as to what the function below does.

Reply:
That much is visible from the types.

Reply:
you are totally correct! 🤦‍♂